coldfront/core/group/tasks.py

```
@@ -0,0 +1,63 @@
+from django.contrib.auth.models import Group
+from coldfront.core.user.models import User
+from coldfront.core.project.models import (
+    Project,
+    ProjectUser,
+    ProjectUserRoleChoice,
+    ProjectUserStatusChoice,
+)
+
+
+def grant_usersupport_global_project_manager() -> None:
+    group_name = "RIS-UserSupport"
+    group = Group.objects.filter(name=group_name).first()
```

@gunnarwustl I would rather throw an exception if the group name is not found since it is necessary for operations. Also, an exception screams loudly that the app is not properly configured.

---

You should now find improved error handling.